src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/DiskSize.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+import { useInstanceTypeVMStore } from '@catalog/CreateFromInstanceTypes/state/useInstanceTypeVMStore';
+import CapacityInput from '@kubevirt-utils/components/CapacityInput/CapacityInput';
+import { getVolumeSnapshotSize } from '@kubevirt-utils/resources/bootableresources/selectors';
+import { formatBytes } from '@kubevirt-utils/resources/vm/utils/disk/size';
+
+const DiskSize: FC = () => {
+  const { instanceTypeVMState, setCustomDiskSize } = useInstanceTypeVMStore();
+
+  const { customDiskSize, pvcSource, volumeSnapshotSource } = instanceTypeVMState;
+
+  const pvcDiskSize =
+    pvcSource?.spec?.resources?.requests?.storage || getVolumeSnapshotSize(volumeSnapshotSource);
+  const sizeData = formatBytes(pvcDiskSize);
```

Comment:
This can cause a bug on certain edge-case, please look at https://github.com/kubevirt-ui/kubevirt-plugin/pull/1999/files